From 4f70ac85787cc9aa47fac1838d5b35337f9be684 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 13 Aug 2017 22:33:50 +0100
Subject: [PATCH] ensurepip: Always install pip/setuptools ignoring installed
 packages

This makes sure that when Python3 is built with a presently installed
Python3, it will still install the setuptools/pip components.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 Lib/ensurepip/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Lib/ensurepip/__init__.py b/Lib/ensurepip/__init__.py
index 25c5567..ce93a56 100644
--- a/Lib/ensurepip/__init__.py
+++ b/Lib/ensurepip/__init__.py
@@ -103,7 +103,7 @@ def bootstrap(*, root=None, upgrade=False, user=False,
             additional_paths.append(os.path.join(tmpdir, wheel_name))
 
         # Construct the arguments to be passed to the pip command
-        args = ["install", "--no-index", "--find-links", tmpdir]
+        args = ["install", "-I", "--no-index", "--find-links", tmpdir]
         if root:
             args += ["--root", root]
         if upgrade:
-- 
2.14.1

